doc/api/console.markdown

```diff
@@ -72,15 +72,15 @@ object. This is useful for inspecting large complicated objects. Defaults to
 - `colors` - if `true`, then the output will be styled with ANSI color codes.
 Defaults to `false`. Colors are customizable, see below.
 
-### console.time(timerName)
+### console.time(label)
```

Comment:
According to MDN history, it's called `timerName` on there since 2012 when that article was created, thought calling it `label` is pretty common too. If we agree on calling it `label`, I can change it on MDN, supposedly. cc: @targos